enhancements/authentication/pod-security-admission-autolabeling.md

```diff
@@ -0,0 +1,290 @@
+---
+title: pod-security-admission-autolabeling
+authors:
+  - "@stlaz"
+reviewers:
+  - "@deads2k"
+  - "@s-urbaniak"
+  - "@soltysh"
+approvers:
+  - "@sttts"
+  - "@mfojtik"
+api-approvers: # in case of new or modified APIs or API extensions (CRDs, aggregated apiservers, webhooks, finalizers)
+  - @sttts"
+creation-date: yyyy-mm-dd
+last-updated: yyyy-mm-dd
+tracking-link: # link to the tracking ticket (for example: Jira Feature or Epic ticket) that corresponds to this enhancement
+  - TBD
+see-also:
+  - "/enhancements/authentication/pod-security-admission.md"
+replaces: []
+superseded-by: []
+---
+
+# Pod Security Admission Autolabeling
+
+## Summary
+
+This enhancement expands the "PodSecurity admission in OpenShift". It introduces
+an opt-in mechanism that allows users to to keep their workloads running when
+Pod Security Admission plugin gets turned on.
+
+## Motivation
+
+We want to adhere with the upstream pod security standards for our workloads but
+we also want to provide our users access to the Security Context Constraints
+(hereinafter SCCs) API that they are already used to. However, each of these
+admission plugins works a bit differently and so there must be a middle-man
+that synchronizes the privileges SCCs provide into privileges that Pod Security
+admission (hereinafter PSa) understands.
+
+### Goals
+
+1. OpenShift clusters can run with the restricted Pod Security admission level by default
+without breaking pod controller workloads
+2. Privileged users can opt-in/opt-out a namespace to and from the Pod Security admission
+
+### Non-Goals
+
+1. Make sure that bare pods (pods w/o pod controllers) continue working
```

Comment:
can we describe why we exclude them?

Reply:
I tried to capture that in a bullet point in this section